app/policies/family_policy.rb

```diff
@@ -131,4 +165,18 @@ def can_view_audit_log?
 
     user.has_hbx_staff_role? && user.person.hbx_staff_role.permission.can_view_audit_log
   end
+
+  # Checks if the user can access the individual market for the current family record.
+  # A user can access the individual market if the primary person of the family has their identity verified,
+  # and the user is either the primary family member, associated with an active broker, or an individual market admin.
+  #
+  # @return [Boolean] Returns true if the user can access the individual market for the current family record, false otherwise.
+  def can_access_individual_market?
+    ridp_verified_primary_person?(record) &&
```

Comment:
- We are initializing FamilyPolicy with family record. 
- Since record itself is family and record available in application policy, do we have to pass record to these methods?